crates/bevy_ecs/src/schedule/executor/simple.rs

```diff
@@ -8,6 +8,8 @@ use crate::{
     world::World,
 };
 
+use super::is_apply_deferred;
```

Comment:
We're already `use`-ing crate::schedule. Mind clustering this with the rest of them?